lmnet/lmnet/utils/signal_handler.py

```diff
@@ -0,0 +1,34 @@
+
+# -*- coding: utf-8 -*-
+# Copyright 2019 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import signal
+
+
+class SignalHandler:
```

Comment:
Making `SignalHandler` class is necessary?
I think it's ok to write signal catch in `train.py`.

Reply:
@iizukak Thanks for the comment.

I created this class to make it reusable somewhere else since it's just a utility function.
Also, I was thinking of decluttering `train.py` as it has so many things going on IMO.
What do you think?